### PR TITLE
replace CommaSeparatedListComponent with pipe

### LINKED ISSUE #3913: Replace CommaSeparatedListComponent with join method or pipe
Component is a bit too much for this. Use `join(', ')` method or our own pipe.

### PR DESCRIPTION
**Pull request by medmedchiheb**
Fixes #3913

### CODE REVIEW

File: src/app/frontend/common/pipes/commaseparated.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({name: 'commaSeparated'})
+export class CommaSeparatedPipe implements PipeTransform {
+  transform(value: any): any {
+    if (!value) {
+      return '';
+    }
+
+    console.log(value);
```
Comment (maciaszczykm) [inline src/app/frontend/common/pipes/commaseparated.ts]: Please remove this.

File: src/app/frontend/common/components/policyrule/template.html [hunk 2]
```diff
@@ -46,7 +48,9 @@
                          i18n>Non-resource URL</mat-header-cell>
         <mat-cell *matCellDef="let rule">
           <ng-container *ngIf="rule.nonResourceURLs">
-            <kd-comma-separated-list [items]="rule.nonResourceURLs"></kd-comma-separated-list>
```
Comment (maciaszczykm) [inline src/app/frontend/common/components/policyrule/template.html]: Old component should be removed.

File: src/app/frontend/common/pipes/commaseparated.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({name: 'commaSeparated'})
+export class CommaSeparatedPipe implements PipeTransform {
+  transform(value: any): any {
```
Comment (maciaszczykm) [inline src/app/frontend/common/pipes/commaseparated.ts]: Return `string`. Param is an array.

**Comment by medmedchiheb** [comment]
@maciaszczykm  i will fix them

**Comment by maciaszczykm** [comment]
Thank you for fixing it and such a quick response!

/lgtm
/approve

**Comment by maciaszczykm** [comment]
Code is not formatted.

**Comment by maciaszczykm** [comment]
Please also merge latest master.

**Comment by maciaszczykm** [comment]
Something went wrong during merge. Can you fix it?

**Comment by maciaszczykm** [comment]
One last thing: `[1] [2] HTML code is not properly formatted. Please run 'npm run fix:frontend'.`.

**Comment by medmedchiheb** [comment]
@maciaszczykm done

**Comment by maciaszczykm** [comment]
Thanks, well done.

/lgtm
/approve

**Pull request merged**